src/MATLAB.jl

```diff
@@ -61,13 +61,15 @@ function nfields(mx::MxArray)
     return mxfields(mx)
 end
 
-function jvariable(mx::MxArray, ty::Type{AbstractString}) 
-    Base.depwarn("jvariable(mx::MxArray,ty::Type{AbstractString}) is 
-    deprecated, use jvariable(mx::MxArray,ty::Type{String}) instead. 
-    We now default to more strict typing on String types", :jvariable)
+function jvalue(mx::MxArray, ty::Type{AbstractString}) 
+    Base.depwarn("jvalue(mx::MxArray,ty::Type{AbstractString}) is 
+    deprecated, use jvalue(mx::MxArray,ty::Type{String}) instead. 
+    We now default to more strict typing on String types", :jvalue)
     return jstring(mx)::String
 end
 
 @deprecate duplicate(mx::MxArray) copy(mx::MxArray)
 
+@deprecate jvariable(mx::MxArray) jvalue(mx::MxArray)
```

Comment:
This doesn't look like all the deprecated signatures. (You probably don't need to specify the signatures).